library/src/scala/CanEqual.scala

```diff
@@ -31,4 +31,13 @@ object CanEqual {
   // source code of these classes
   given canEqualSeq[T, U](using eq: CanEqual[T, U]): CanEqual[Seq[T], Seq[U]] = derived
   given canEqualSet[T, U](using eq: CanEqual[T, U]): CanEqual[Set[T], Set[U]] = derived
+
+  // The next three typeclass instances can also go into the companion objects of classes Option and Either.
+  // For now they are here in order not to have to touch the source code of these classes
```

Comment:
I think you could update the comment above to say "The next five definitions ..." instead of adding a new comment.

Reply:
@smarter Sure, I've just updated it. Thank you!